ui-new/packages/tidb-dashboard-lib/src/apps/Overview/components/MetricsItems.tsx

```diff
@@ -0,0 +1,398 @@
+const MetricsItems = [
```

Comment:
This is not a component, it's a data or config, maybe better to put it in the `data` folder.